### DISCUSSION TITLE
Add `flip_h` and `flip_v` to `Button` icon

### DISCUSSION THREAD

**Discussion by acgc99** [godotengine/godot-proposals#8207] [GUI]
This will reduce the use of rotated icons (separateed image files) if required. It would work like `TextureRect`.

**Comment by Mickeon** [comment]
I do not feel much interest for this. What's the use case?

**Reply by acgc99** [reply]
Just giving the option for rotating the icon would reduce the number of icon files. The main use would be for arrows I suppose.

**Reply by samuelfine** [reply]
Checking in a year after this original request - I could definitely use this, specifically for the reason given here. It will allow for quick reuse of textures as `Button` icons without requiring additional unique assets, similar to how it is used in `Sprite`s today.

**Reply by julia-maliauka** [reply]
I found this post while looking for this option, so count me as another person who would use it. I believe TextureButton has a flipH/flipV  option, so something like this would be great for regular buttons, especially for things like arrows and so on.

**Reply by Helianthropy** [reply]
Yep, reducing number of assets for mirroring icons between two configurable characters on screen, for instance. I’m coming at this from the ItemList button type. 🙂

**Reply by klicktock** [reply]
It's also useful for Arabic and right-to-left languages. "Back" button needs to be flipped.

**Comment by Synzorasize** [comment]
Opened a PR for this: godotengine/godot#122030

**Comment by ObedH** [comment]
This would be helpful for me as well.
